wremnants/muon_calibration.py

```diff
@@ -538,6 +539,8 @@ def define_corrected_reco_muon_kinematics(df, muons="goodMuons", kinematic_vars
     return df
 
 def transport_smearing_weights_to_reco(resultdict, procs, nonClosureScheme = "A-M-separated"):
+    time0 = time.time()
```

Comment:
Is this particularly slow?

Reply:
Not really, when I was working on the optimization I was interested in the timing and I think it does not harm leaving it here